backend/LexBoxApi/GraphQL/LexMutations.cs

```diff
@@ -42,12 +43,27 @@ public LexMutations(LoggedInContext loggedInContext)
         return await dbContext.Projects.Where(p => p.Id == input.ProjectId).FirstAsync();
     }
 
+    [Error<NotFoundException>]
+    [Error<DbError>]
+    [UseMutationConvention]
+    public async Task<ProjectUsers> ChangeProjectMemberRole(ChangeProjectMemberRoleInput input,
+        LexBoxDbContext dbContext)
+    {
+        var projectUser = await dbContext.ProjectUsers.FirstOrDefaultAsync(u => u.ProjectId == input.ProjectId && u.UserId == input.UserId);
+        if (projectUser is null) throw new NotFoundException("Project member not found");
+        projectUser.Role = input.Role;
+        await dbContext.SaveChangesAsync();
+        return projectUser;
+    }
+
     [Error<NotFoundException>]
     [Error<DbError>]
     [UseMutationConvention]
     public async Task<Project> ChangeProjectName(ChangeProjectNameInput input,
         LexBoxDbContext dbContext)
     {
+        if (input.Name.IsNullOrEmpty()) throw new RequiredException("Project name cannot be empty");
```

Comment:
because you're throwing a new exception here you should add an attribute for it on this method, just like the  NotFoundException, it's so that we can tell HotChocolate which exceptions are errors and should be returned to the user vs not.